.eslintignore

```diff
@@ -1,4 +1,2 @@
-.vscode/*
+.*
 coverage/*
-.hlx
-dist
```

Comment:
So `dist` was removed from the ignore list?

Reply:
oh...